tests/expected_report.txt

```diff
@@ -24,13 +24,14 @@ tests/input/return_type_on_methods.php                17      0
 tests/input/semicolon_spacing.php                     3       0
 tests/input/static-closures.php                       1       0
 tests/input/test-case.php                             8       0
+tests/input/trailling_comma_on_array.php              1       0
```

Comment:
Typo: trailing 🤓

Also, the file name should also use dashes for consistency with other file names.

Reply:
> Typo: trailing 
> 
> Also, the file name should also use dashes for consistency with other file names.

that's mine

Reply:
I was thinking if I need to use dashes or underlines

Reply:
Right. Might as well leave it as long as the typo is fixed :)